### PR TITLE
tests: remove unstable warnings, fix test script

### PR DESCRIPTION
**Pull request by therve**
This removes the error in go 1.13, plus a few warnings

### CODE REVIEW

**Review by zippolyte — DISMISSED** [review]

File: tests/scenarios.go [hunk 1]
```diff
@@ -305,11 +309,7 @@ func GetRequestsUndo(ctx gobdd.Context, operationID string) (func(interface{}) f
 				in[i] = object
 			}
 			request := undoOperation.Call(in)[0]
-			result := request.MethodByName("Execute").Call(nil)
-
-			if result[len(result)-1].Interface() != nil {
-				fmt.Printf("error in undo %v", result[len(result)-1])
-			}
+			request.MethodByName("Execute").Call(nil)
```
Comment (jirikuncar) [inline tests/scenarios.go:312]: we should try to add `t.Logf("error in undo %v", result[len(result)-1])`

File: tests/scenarios.go [hunk 2]
```diff
@@ -183,9 +183,11 @@ func (s GivenStep) RegisterSuite(suite *gobdd.Suite) {
...
 		// Enable unstable operation
 		configInterface := reflect.Indirect(clientInterface.(reflect.Value)).Addr().MethodByName("GetConfig").Call(nil)[0]
-		configInterface.MethodByName("SetUnstableOperationEnabled").Call([]reflect.Value{
-			reflect.ValueOf(s.OperationID), reflect.ValueOf(true),
-		})
+		if configInterface.MethodByName("IsUnstableOperation").Call([]reflect.Value{reflect.ValueOf(s.OperationID)})[0].Bool() {
+			configInterface.MethodByName("SetUnstableOperationEnabled").Call([]reflect.Value{
+				reflect.ValueOf(s.OperationID), reflect.ValueOf(true),
+			})
+		}
```
Comment (jirikuncar) [inline tests/scenarios.go:190]: 👍

File: tests/scenarios.go [hunk 3]
```diff
@@ -289,9 +291,11 @@ func GetRequestsUndo(ctx gobdd.Context, operationID string) (func(interface{}) f
...
 			// enable unstable operation
 			configInterface := reflect.Indirect(undoClientInterface.(reflect.Value)).Addr().MethodByName("GetConfig").Call(nil)[0]
-			configInterface.MethodByName("SetUnstableOperationEnabled").Call([]reflect.Value{
-				reflect.ValueOf(undo.Undo.OperationID), reflect.ValueOf(true),
-			})
+			if configInterface.MethodByName("IsUnstableOperation").Call([]reflect.Value{reflect.ValueOf(undo.Undo.OperationID)})[0].Bool() {
+				configInterface.MethodByName("SetUnstableOperationEnabled").Call([]reflect.Value{
+					reflect.ValueOf(undo.Undo.OperationID), reflect.ValueOf(true),
+				})
+			}
```
Comment (jirikuncar) [inline tests/scenarios.go:298]: 👍

**Review by jirikuncar — CHANGES_REQUESTED** [review]
please keep logging of failed undo methods

File: tests/scenarios.go (hunk 1 above)
Comment (therve) [reply tests/scenarios.go:312]: I don't have access to `t` here, should I just use the log package?

Comment (zippolyte) [reply tests/scenarios.go:312]: using log will print to stderr, which will fail trigger a failure in gotestsum, not sure we want that

**Review by jirikuncar — APPROVED** [review]

**Pull request merged**